threat-model/readme.md

```diff
@@ -99,85 +93,159 @@ See [section 2. First Parties, Embedded Parties and Delegated Parties](#2-First-
     1. Exception that 1.2.1.2 would be partitioned delegated party context.
 
 
 #### 1.3.2. Capabilities
 
-
-
 1. The same capabilities as First Parties ([section 1.2.2](#122-Capabilities))
 2. Delegated parties can modify client assets through the proposed APIs, which may include overwriting/modifying data provided by the first party or other delegated parties.
 
 
 #### 1.3.3. Mitigations
 
-
-
 1. The same mitigations as Third Parties ([section 1.2.3](#123-Mitigations))
 2. The client should prevent the ability of a delegated party to overwrite/modify data provided by the first party or other delegated parties. If this is not possible, the client should allow the first party to control and limit a delegated party's ability in this manner. (See [section 2. First Parties, Embedded Parties and Delegated Parties](#2-First-Parties-Embedded-Parties-and-Delegated-Parties) for more details.)
 
 
-### 1.4. Aggregator
+### 1.4 Helper Parties and Helper Party Networks
 
-An aggregator is an individual party which participates in an aggregation protocol. We assume that some (proposal specific) subset of the aggregators act honestly and do not collude with other aggregators or any other parties. Here we outline assets and capabilities assuming that an aggregator colludes up to, but not exceeding, that threshold.
+Helper parties are a class of party (i.e., companies, organizations) who participate in a protocol in order to instantiate a private computation system. There are currently two types of helper parties proposed, _aggregators_ and _coordinators_. We outline the properties on each type directly (as opposed to on the class itself.)
 
+We define a _helper party network_ as a collection of specific parties who are acting as helper parties. We assume that an attacker can control some (proposal specific) subset of the helper party network, and that the remaining helper parties act honestly (e.g., they do not collude with other helper parties or any other parties.) Here we outline assets and capabilities in the presence of this attacker.
 
-#### 1.4.1. Assets
+All parties in a helper party network should be known a priori and web platform vendors should be able to evaluate risk of an attacker that is more powerful than our assumption, e.g., the attacker is able to control more than the (protocol specific) subset of helper parties in the network.
 
 
+### 1.5. Aggregator Helper Party
+
+An aggregator is type of helper party which participates in a helper party network to instantiate an MPC based private computation system. Aggregators receive secret shares of the input data and interact with the other aggregators in the helper party network to perform the aggregation. See [section 4.1 Multi-pary Computation](#41-Multi-party-Computation) for more details.
 
-1. Unencrypted individual share.
-2. All the assets from first and third parties.
-3. Shares of the output of the aggregation protocol.
-4. Identity of other aggregators.
 
+#### 1.5.1. Assets
```

Comment:
Maybe we can add this in a follow-up but something missing in bot the aggregator assets and coordinator assets is the _state_ of the privacy budget accountant which you reference later on in the doc.

Reply:
Yes, I think we need a whole issue / PR to deal with privacy budgeting. It is not at all sufficiently addressed, and, depending on how it works, could present all types of issues.

Reply:
added to https://github.com/patcg/docs-and-reports/issues/16